cmd/gapit/report.go

```diff
@@ -33,11 +33,7 @@ type reportVerb struct{ ReportFlags }
 
 func init() {
 	verb := &reportVerb{
-		ReportFlags: ReportFlags{
-			CommandFilterFlags: CommandFilterFlags{
-				Context: -1,
-			},
-		},
+		ReportFlags: ReportFlags{},
```

Comment:
this is unnecessary, unspecified fields are initialized to their zero value.

Reply:
Done.